scripts/insert_into_party.py

```diff
@@ -0,0 +1,58 @@
+import sys
```

Comment:
Scripts need better naming - "insert_into_party" doesn't give any context of _what_ it inserts and _why_. If it's just for testing it should be clearly separated from scripts used for deployment/rollback.

Reply:
Renamed to create_example_business_data.  I'll put an explanation of what the script does in a docstring as moving it into a deeper folder would mean having to mess around with paths some more which I don't really want to do (as the first 4 lines of this are messy enough as it is)